src/node_options.cc

```diff
@@ -429,6 +429,11 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "write warnings to file instead of stderr",
             &EnvironmentOptions::redirect_warnings,
             kAllowedInEnvironment);
+  AddOption("--set-output-dir",
+            "set dir for all output files"
+            " (default: current working directory of Node.js process)",
```

Comment:
```suggestion
            " (default: current working directory)",
```

Because its more consistent, and what a working directory is pretty obvious.

```
~ % node --help | grep -C1 working
                              specified, write the profile to the
                              current working directory.
  --cpu-prof-dir=...          Directory where the V8 profiles
--
--
                              specified, write the profile to the
                              current working directory.
  --heap-prof-dir=...         Directory where the V8 heap profiles
--
--
  --report-directory=...      define custom report pathname.
                              (default: current working directory of
                              Node.js process)
```